pkgs/servers/pulseaudio/default.nix

```diff
@@ -31,23 +35,27 @@
 
 stdenv.mkDerivation rec {
   pname = "${if libOnly then "lib" else ""}pulseaudio";
-  version = "14.2";
+  version = "15.0";
 
   src = fetchurl {
     url = "http://freedesktop.org/software/pulseaudio/releases/pulseaudio-${version}.tar.xz";
-    sha256 = "sha256-ddP3dCwa5EkEmkyIkA5FS4s1DsqoxUTzSIolYqn/ZvE=";
+    sha256 = "sha256-pAuIejupjMJpdusRvbZhOYjxRbGQJNG2VVxqA8nLoaA=";
   };
 
+  patches = [
+    ./add-option-for-installation-sysconfdir.patch
```

Comment:
Interestingly, this patch has been sticking around since 33784354b96055c4b4d40fc12aa88b3862b6048e from 2020 but has not been used for the main pulseaudio package so far, just for `pulseaudio-hsphfpd`.

Please add a short comment for why this patch is necessary.